test/agoneum/web/controllers/registration_controller_test.exs

```diff
@@ -1,30 +1,47 @@
 defmodule Agoneum.Web.RegistrationControllerTest do
   use Agoneum.Web.ConnCase
 
-  alias Agoneum.Account
+  describe "UI registration" do
```

Comment:
I'm for splitting the tests out, but wondering if we actually need a `controllers` directory? How would you feel about making these more inline with `request` / `feature` specs (as far as structure goes) ?